.travis.yml

```diff
@@ -18,9 +18,17 @@ stages:
   - test
   - name: release
     if: branch =~ /^(\d+\.x|master|next|beta)$/ AND type IN (push)
+  - name: update-prettier
+    if: branch =~ ^greenkeeper/prettier
 
 jobs:
   include:
+    - stage: update-prettier
+      script:
+        - git checkout $TRAVIS_BRANCH
+        - npm run prettier
+        # commit changes and push back to branch on GitHub. If there are no changes then exit without error
+        - 'git commit -a -m "style: prettier" && git push "https://${GH_TOKEN}@github.com/$TRAVIS_REPO_SLUG" ${TRAVIS_BRANCH} || true'
```

Comment:
One thing that I'd love to do is to cancel the the build if there was an update. The new commit will cause a new build anyway. I've enabled the "Auto cancel branch builds" setting on travis, but I'm not sure if that will cancel the current build as it already started?

Reply:
Yea, I think that's what cancelation is intended to do, e.g. when someone pushes a new commit while the build is in progress.